app/models/applicant.rb

```diff
@@ -22,4 +22,18 @@ def status_progress
       self.status = 'In Progress'
     end
   end
+
+  def approved?(applicant_id)
+    application_pets = PetApplicant.where(applicant_id: applicant_id)
+    application_pets.all? do |pet|
+        pet.status == 'approve'
```

Comment:
This should be using Active Record not ruby enumerables. So you could do application_pets.where(status: 'approved') to achieve the same result and it is MUCH more efficient than using a ruby enumerable

Reply:
ahhhh. Okay this is where I need to start switching my brain to active record mode instead of Ruby. I keep thinking Ruby on how to solve these. Definitely need to work on making that adjustment.